Us 17 redirection

MiklCouF opened the pull request.

MiklCouF committed:
adding redirect link if user is login or not (using the token in localstorage)

MiklCouF committed:
4Merge branch 'main' of https://github.com/WildCodeSchool-2024-02/JS-Nantes-RamCSS-P3-PoPote into US_17_redirection

MiklCouF committed:
redirection to connection for visitor not logged

TheoDegorre reviewed (approved):
Tout est beau, tout est propre

MrLootman commented on client/src/components/NavBar.jsx:
Attention : La variable isLogged n'est pas une variable d'état, ce qui signifie que le rafraîchissement dynamique ne pourra pas se faire. Autrement dit, isLogged est forcément false dans le return du composant NavBar.

MiklCouF committed:
Merge branch 'main' into US_17_redirection

MrLootman commented on client/src/components/NavBar.jsx:
```diff
@@ -3,14 +3,26 @@ import { NavLink, useLocation } from "react-router-dom";
   const location = useLocation();
 
+  const token = localStorage.getItem("token");
+  let isLogged = "false";
```
c'est sus

MrLootman commented on client/src/components/NavBar.jsx:
```diff
@@ -3,14 +3,26 @@ import { NavLink, useLocation } from "react-router-dom";
+      <NavLink
+        className="nav-link"
+        to={isLogged ? "/popote/" : "/connexion"}
+        aria-label="Page d'acceuil"
```
accueil*

MrLootman reviewed (dismissed):
Merci de lire les commentaires

MiklCouF committed:
Update NavBar.jsx

Change Accueil syntax, with craft code, like the ancient time when Chatgt doesn't exist.
